Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM dmstraub/gramps:5.1.4
 WORKDIR /app
 ENV PYTHONPATH="${PYTHONPATH}:/usr/lib/python3/dist-packages"
 
+RUN apt-get update && apt-get install -y libpq-dev postgresql-client postgresql-client-common python3-psycopg2 && rm -rf /var/lib/apt/lists/*
```

Comment:
Can you please merge these two lines so apt doesn't have to fetch updates twice?

Reply:
Also done.